content/blog/using-apache-spark-dataframes-for-processing-of-tabular-data.md

```diff
@@ -0,0 +1,362 @@
+---
+title: Using Apache Spark DataFrames for Processing of Tabular Data
+date: 2021-10-14T05:35:43.497Z
+priority: 4
+author: Carol McDonald
+authorimage: /img/Avatar1.svg
+tags:
+  - HPE Ezmeral Data Fabric
+  - MapR
+  - Apache Spark
+---
+```
+"authorDisplayName": "Carol McDonald",
+"category": "apache-spark",
+"publish": "2015-06-24T07:00:00.000Z"
+```
+
+**Editor’s Note:** MapR products and solutions sold prior to the acquisition of such assets by Hewlett Packard Enterprise Company in 2019 may have older product names and model numbers that differ from current solutions. For information about current offerings, which are now part of HPE Ezmeral Data Fabric, please visit <https://www.hpe.com/us/en/software/ezmeral-data-fabric.html>
+
+This post will help you get started using Apache Spark DataFrames with Scala on the MapR Sandbox. The <a target='\_blank'  href='http://spark.apache.org/docs/latest/sql-programming-guide.html#datasets-and-dataframes'>new Spark DataFrames API</a> is designed to make big data processing on tabular data easier.
```

Comment:
Hi Nathan, 

Last year the MapR Sandbox was one of the pending migration items from MapR.com. And I believe that this blog post was put on hold because of this pending issue.

Do you confirm that the MapR Sandbox is no longer a pending migration item from MapR.com and it is now the "Development Environment for HPE Ezmeral Data Fabric"?

If it is confirmed, I suggest you add a note in this sentence to something like this:

This post will help you get started using Apache Spark DataFrames with Scala on the MapR Sandbox (now known as the Development Environment for HPE Ezmeral Data Fabric).

What do you think?

Reply:
done
They haven't had me redirect the sandbox part of the old mapr website, but yes, it looks like the container option is the replacement for the sandbox per the install instructions.